src/System.Drawing.Common/src/System/Drawing/Bitmap.NotSerializable.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Bitmap itself isn't declared ISerializable: http://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/Bitmap.cs,37

Its parent type `System.Drawing.Image` is. You should remove this file